apps/storybook-e2e/src/e2e/components/progress-bar/vl-progress-bar.stories.cy.ts

```diff
@@ -5,94 +5,23 @@ const progressBarNumericUrl =
 const progressBarFocusedUrl =
     'http://localhost:8080/iframe.html?id=components-progress-bar--progress-bar-focused&viewMode=story';
 
-const changeActiveStep = (stepNumber: number) => {
-    cy.get('vl-progress-bar').invoke('attr', 'data-vl-active-step', stepNumber);
-};
-
-const shouldSetSteps = () => {
-    cy.get('vl-progress-bar').shadow().find('.vl-progress-bar__step').should('have.length', 3);
-};
-
-const shouldHaveActiveStep = (stepNumber: number) => {
-    cy.get('vl-progress-bar')
-        .shadow()
-        .find('.vl-progress-bar__step')
-        .eq(stepNumber - 1)
-        .should('have.class', 'vl-progress-bar__step--active');
-};
-
-const shouldSetCorrectActiveStep = () => {
-    shouldHaveActiveStep(1);
-    changeActiveStep(2);
-    shouldHaveActiveStep(2);
-    changeActiveStep(3);
-    shouldHaveActiveStep(3);
-};
-
-const shouldHaveVisibleTooltipForStep = (stepNumber: number) => {
-    cy.get('vl-progress-bar')
-        .shadow()
-        .find('.vl-progress-bar__step')
-        .eq(stepNumber - 1)
-        .find('button.vl-progress-bar__bullet')
-        .next()
-        .should('have.attr', 'aria-hidden', 'false');
-};
-
-describe('story vl-progress-bar default', () => {
-    it('should set steps', () => {
-        cy.visit(progressBarUrl);
-
-        shouldSetSteps();
-    });
-
-    it('should set correct active step', () => {
+describe('story vl-progress-bar - default', () => {
+    it('should display story', () => {
         cy.visit(progressBarUrl);
-
-        shouldSetCorrectActiveStep();
+        cy.get('vl-progress-bar');
```

Comment:
Kan je hier checken of er een shadow dom is? Deze test gaat altijd slagen ook al wordt de vl-progress-bar niet gerenderd.